textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -43,36 +44,106 @@ public FuzzyTokenizer(String phrase) {
     }
 
     /**
+     * When called returns a single-member list containing most-likely tokenization of phrase
```

Comment:
For line 42 `public FuzzyTokenizer(String phrase`, rename `phrase` to `queryString` since `phrase` has a special meaning in other modules.